### PR TITLE
Fix CI build with core

### PR DESCRIPTION
**Pull request by andiwand**
Fixes https://github.com/opendocument-app/OpenDocument.droid/issues/336

### CODE REVIEW

File: app/src/main/java/at/tomtasche/reader/background/PdfLoader.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public void loadSync(Options options) {
             File cacheDirectory = AndroidFileCache.getCacheDirectory(cacheFile);
 
             pdf2htmlEX pdfConverter = new pdf2htmlEX(context).setInputPDF(cacheFile);
-            pdfConverter.setOutline(false).setBackgroundFormat("jpg").setDRM(false);
+            pdfConverter.setDRM(false);
```
Comment (andiwand) [inline app/src/main/java/at/tomtasche/reader/background/PdfLoader.java:50]: @TomTasche @ViliusSutkus89 do you know what should be done here? It complained about the functions being not existent and I just removed the calls for now.

Comment (TomTasche) [reply app/src/main/java/at/tomtasche/reader/background/PdfLoader.java:50]: if I remember correctly that disables the overview of pages on the left side, so it was quite important for us! can you remember why it was removed? @ViliusSutkus89

Comment (andiwand) [reply app/src/main/java/at/tomtasche/reader/background/PdfLoader.java:50]: Maybe it was just replaced with a different method?

What confused me is why does this break now? Are we not pinning the version correctly of this library?

Comment (TomTasche) [reply app/src/main/java/at/tomtasche/reader/background/PdfLoader.java:50]: the outline is now visible (black bar on the left in screenshot), which is not what we want. we can't release like that :disappointed: help @ViliusSutkus89 
image

Comment (ViliusSutkus89) [reply app/src/main/java/at/tomtasche/reader/background/PdfLoader.java]: Hey, is this problem still relevant? I've tried running current main version and outline isn't there

Comment (andiwand) [reply app/src/main/java/at/tomtasche/reader/background/PdfLoader.java]: I actually didn't try and just made it compile. Did you try @TomTasche ?

Comment (TomTasche) [reply app/src/main/java/at/tomtasche/reader/background/PdfLoader.java]: I did, all good!

**Pull request merged**